### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [narwhrl/narwhrl.github.io#3] [Giscus]
# index

https://pan.startrekcn.cn/

**Comment by narwhrl** [comment]
<h2>更新日志 2023.01.25</h2>

替换Valine, 使用Giscus评论系统.

**Reply by rinntaya** [reply]
写错了吧，是giscus(oﾟvﾟ)ノ

**Reply by narwhrl** [reply]
确实是hhhh，当时 Giscus&Disqus 没分清楚🤣

**Comment by cloudloong-debug** [comment]
昨天看到贵站自建网盘中有深空九站的资源，今天想来下载却发现没有了，请问何时可以再次开放下载，谢谢！

**Reply by cloudloong-debug** [reply]
昨天可以访问自建网盘的根目录，资源很多，今天只能看到五部剧集😂求资源🙏🙏

**Reply by narwhrl** [reply]
其实一直没打算开放过....也不知道你是怎么访问到根目录的...😂
DS9和VOY有些集的字幕压出来是有问题的，所以就没打算提供下载
更推荐选择幻翔或是网飞版本，或是直接去[https://video.startrekcn.cn](https://video.startrekcn.cn)观看

**Reply by cloudloong-debug** [reply]
翻历史记录重新确认了一下，2-12周日下午17:40左右确实访问到根目录😂😂然后访问了：“ /电视剧/星际迷航系列/[1993][DS9][深空九号][幻翔]/S7......”诸如此类，没有下载就关闭了网页，刚才打开历史记录已经提示“failed get storage: can't find storage with rawPath: /电视剧/......"
特此报告🙋‍

**Reply by cloudloong-debug** [reply]
也许，网站存在漏洞，当时浏览页标签标注的不是现在的”ST剧集储存库“，而是”Narwhrl的储存库“，特别说明🙋‍

**Reply by narwhrl** [reply]
也许是，不过已经无法复现了。不过你要下载的话也可以自行前往<a href="https://drive.narwh.ch">https://drive.narwh.ch</a>进行下载。

**Reply by cloudloong-debug** [reply]
Thanks!

**Comment by NCC-1701-A** [comment]
能否开放https://drive.narwh.ch/的访客访问？我想下载下一代系列第一季至SD卡观看

**Reply by narwhrl** [reply]
[星际迷航吧](https://tieba.baidu.com/f?kw=%E6%98%9F%E9%99%85%E8%BF%B7%E8%88%AA&fr=index#)置顶帖自行下载

**Reply by NCC-1701-A** [reply]
非常感谢！

**Comment by hoshiizumiya** [comment]
是否有外挂字幕，能否单独打包放到网盘中，或者上传到各大字幕站里，例如字幕库, subhd等

**Comment by GaryWank** [comment]
神童， 还做不做字幕？

**Comment by GaryWank** [comment]
神童不做字幕了 ？

**Comment by 404reposter** [comment]
failed get objs: failed to list objs: The request has been throttled
有人反馈过上述问题吗，几个资源都显示这个

**Reply by OndayX** [reply]
刚反馈+1

**Comment by OndayX** [comment]
[20250224]点进任何一个文件夹都显示：failed get objs: failed to list objs: Please try again later.

**Comment by richyhu** [comment]
为什么所有都没了
